app/models/concerns/runner_user.rb

```diff
@@ -0,0 +1,18 @@
+require 'active_support/concern'
+
+module RunnerUser
+  extend ActiveSupport::Concern
+
+  included do
+    def s3_bytes_used(since: 100.years.ago)
+      runs.pluck(:s3_filename).sum(0) do |s3_filename|
+        o = $s3_bucket_internal.object("splits/#{s3_filename}")
```

Comment:
`o` - let's think up a better name for this variable :)

Reply:
Could always just go with `run_object`.